src/ExpressionEvaluator/Core/Source/ResultProvider/Helpers/DynamicFlags.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.ExpressionEvaluator
+{
+    /// <remarks>
+    /// WARNING: Mutable struct.
+    /// </remarks>
+    internal struct DynamicFlags
```

Comment:
Which sentiment are you +1’ing?

From: Kevin Halverson [mailto:notifications@github.com]
Sent: Wednesday, April 15, 2015 11:46 AM
To: dotnet/roslyn
Cc: Andrew Casey (ROSLYN)
Subject: Re: [roslyn] Add support for "dynamic" in the result provider (#1747)


In src/ExpressionEvaluator/Core/Source/ResultProvider/Helpers/DynamicFlags.cs<https://github.com/dotnet/roslyn/pull/1747#discussion_r28449902>:

> @@ -0,0 +1,41 @@

> +// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.

> +

> +namespace Microsoft.CodeAnalysis.ExpressionEvaluator

> +{

> +    /// <remarks>

> +    /// WARNING: Mutable struct.

> +    /// </remarks>

> +    internal struct DynamicFlags

+1...I assume the overhead of these objects wouldn't be huge in normal apps (no dynamic info).

—
Reply to this email directly or view it on GitHub<https://github.com/dotnet/roslyn/pull/1747/files#r28449902>.